src/main.rs

```diff
@@ -1,3 +1,62 @@
-fn main() {
-    println!("Hello, World!");
+use anyhow::Result;
```

Comment:
Not too worried about this since it does the job. But It'd be better to use the built-in `Result<T, E>` enum

Reply:
This is neater given I can generically handle different type of error, but I get your point